.github/workflows/package-workflow.yml

```diff
@@ -25,13 +27,39 @@ jobs:
         version: 12.x
 
     - name: 'Build the standard bundle'
-      run: |
-        set -ex
-        node ./scripts/run-yarn.js build:cli
+      run: node ./scripts/run-yarn.js build:cli
 
     - name: 'Build the dist directory'
-      run: |
-        ./scripts/build-dist.sh
+      run: ./scripts/build-dist.sh
 
     - name: 'Build the deb archive'
       uses: ./scripts/actions/build-deb
+
+  windows-installer:
+    name: 'Windows Installer'
+    runs-on: windows-2016
```

Comment:
windows-2019 doesn't seem to have WiX installed... Will follow up with GitHub support about it.